### ISSUE TITLE
@wdio/local-runner@7.0.2 saucelab ERROR @wdio/runner: Error: Failed to create session.

### ISSUE THREAD

**Issue by borracciaBlu** [webdriverio/webdriverio#6413]
**Dev Dependencies**

```
  "devDependencies": {
    "@wdio/cli": "^7.0.1",
    "@wdio/local-runner": "^7.0.1",
    "@wdio/mocha-framework": "^7.0.0",
    "@wdio/sauce-service": "^7.0.2",
    "@wdio/spec-reporter": "^7.0.1",
    "@wdio/sync": "^7.0.1",
    "assert": "^2.0.0",
    "banner-cli": "^0.14.3",
    "browserify": "^17.0.0",
    "eslint": "^7.19.0",
    "gh-pages": "^3.1.0",
    "mocha": "^8.2.1",
    "nyc": "^15.1.0",
    "uglify-js": "^3.12.7"
  }
```

**Repo examples:**
https://github.com/borracciaBlu/dh-addeventlistener
https://github.com/borracciaBlu/dh-classlist

**Config of WebdriverIO**
https://github.com/borracciaBlu/dh-classlist/blob/master/wdio.conf.js

**Describe the bug**
```
[0-0] 2021-02-12T04:06:41.852Z ERROR webdriver: Request failed with status 401 due to unknown error: failed serving request POST /wd/hub/session: Unauthorized
[0-0] 2021-02-12T04:06:41.852Z ERROR webdriver: unknown error: failed serving request POST /wd/hub/session: Unauthorized
    at Object.getErrorFromResponseBody (/Code/dh-classlist/node_modules/webdriver/build/utils.js:189:12)
    at WebDriverRequest._request (/Code/dh-classlist/node_modules/webdriver/build/request.js:168:31)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.startWebDriverSession (/Code/dh-classlist/node_modules/webdriver/build/utils.js:63:20)
    at async Function.newSession (/Code/dh-classlist/node_modules/webdriver/build/index.js:56:45)
    at async Object.exports.remote (/Code/dh-classlist/node_modules/webdriverio/build/index.js:56:22)
    at async Runner._startSession (/Code/dh-classlist/node_modules/@wdio/runner/build/index.js:198:56)
    at async Runner._initSession (/Code/dh-classlist/node_modules/@wdio/runner/build/index.js:160:25)
    at async Runner.run (/Code/dh-classlist/node_modules/@wdio/runner/build/index.js:83:19)
[0-0] 2021-02-12T04:06:41.852Z ERROR @wdio/runner: Error: Failed to create session.
Session request was not authorized because you either did provide a wrong access key or tried to run in a region that has not been enabled for your user. If have registered a free trial account it is connected to a specific region. Ensure this region is set in your configuration (https://webdriver.io/docs/options.html#region).
    at Object.startWebDriverSession (/Code/dh-classlist/node_modules/webdriver/build/utils.js:68:15)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[0-0] FAILED in MicrosoftEdge - /test/specs/browser-test.e2e.js
```
**To Reproduce**
after cloning `dh-classlist` or `dh-addeventlistener`
env SAUCE_USERNAME="[username]" SAUCE_ACCESS_KEY="[key]" npm run saucelab

**Expected behavior**
Run without 401

**Additional context**
switched back to `@wdio/local-runner@7.0.1` and it does work fine. Same issue on `gh-actions`.

**Comment by alexdinari** [comment]
I've experienced the same exact issue, switching back to also worked for me `@wdio/local-runner@7.0.1`.

**Comment by christian-bromann** [comment]
Fixed in #6413

**Comment by borracciaBlu** [comment]
working like a charm! Tnx @christian-bromann
